stingray/modeling/tests/test_parameterestimation.py

```diff
@@ -161,8 +161,11 @@ def test_sampler_runs(self):
         pe = ParameterEstimation()
         if os.path.exists("test_corner.pdf"):
             os.unlink("test_corner.pdf")
-        sample_res = pe.sample(self.lpost, [2.0], nwalkers=100, niter=10,
-                               burnin=50, print_results=True, plot=True)
+        with warnings.catch_warnings():
```

Comment:
if you want to use it, astropy has a convenience function to catch warnings in tests, `from astropy.tests.helper import catch_warnings, it's`from astropy.tests.helper import catch_warnings`
More info for various tools we have: http://docs.astropy.org/en/stable/testhelpers.html#astropy-testing-tools

Reply:
Right, thanks!

Reply:
I don't see how to reproduce this mechanism though, and ignore only the `RuntimeWarnings`.